requirements.in

```diff
@@ -1,7 +1,7 @@
 gcloud==0.18.3
 google-cloud-bigquery==1.24.0
 google-cloud-storage==1.26.0
-Jinja2==2.11.1
+Jinja2==2.10.3
```

Comment:
There is a conflict with `apache-airflow` which has `jinja2<2.11.0,>=2.10.1` as requirement